src/helpers/sendgridMailTemplate.js

```diff
@@ -0,0 +1,12 @@
+import tokenGenerator from './tokenGenerator';
+
+export default {
+  resetPassword: (to, from) => ({
+    to,
+    from,
+    subject: 'Password Reset',
+    html: `<p>You are receiving this because you (or someone else) have requested the reset of the password,<br>
+     Click on the reset link bellow to complete the process<br>
+     <a href='http://localhost:3000/api/v1/auth/reset/${tokenGenerator({ email: to }, { expiresIn: '2h' })}' target='_blank'>Reset Password</a></p>`
```

Comment:
@luc-tuyishime, I propose the redirect URL be defined in an .env file so that we don't have to change this file in case we want to use another URL, so it maybe something like `APP_URL` and the you use it  this file

Reply:
makes sense. Let's do that

Reply:
Changes applied